docs/accordion.md

```diff
@@ -0,0 +1,93 @@
+# Accordion
+### Usage
+
+```jsx
+import { Accordion } from 'radiance-ui';
+
+<Accordion
```

Comment:
for these example codesnippets, I've been writing the almost the exact same code as what gets rendered in the story below it with some modifications (i.e. don't need show the exact function definition of `onClick` etc.

You can check out the example on the existing stories